tests/components/sensor/common.py

```diff
@@ -141,3 +107,16 @@ async def async_added_to_hass(self) -> None:
         self._values["native_unit_of_measurement"] = (
             last_sensor_data.native_unit_of_measurement
         )
+
+
+def get_mock_sensor_entities() -> dict[str, MockSensor]:
+    """Get mock sensor entities."""
+    return {
+        device_class: MockSensor(
+            name=f"{device_class} sensor",
+            unique_id=f"unique_{device_class}",
+            device_class=device_class,
+            native_unit_of_measurement=UNITS_OF_MEASUREMENT.get(device_class),
+        )
+        for device_class in ("none", *SensorDeviceClass)
```

Comment:
`"none"` is not a valid device class.

Reply:
Right, but it doesn't look correct. Why do we need it?

Reply:
It is used by the `test_get_trigger_capabilities_none` and `test_get_condition_capabilities_none` test.

Reply:
Can we use the MockEntity directly in those tests and create an entity without a device class instead of creating an invalid entity here?

Reply:
Sure